crates/codegen/runtime/cargo/src/runtime/mod.rs

```diff
@@ -69,3 +72,14 @@ pub mod text_index {
     use metaslang_cst::text_index;
     pub use text_index::{TextIndex, TextRange, TextRangeExtensions};
 }
+
+pub mod graph_builder {
```

Comment:
If we are adding/publishing this new API publicly, I think we should also discuss user docs/examples along with it.

Happy to discuss f2f in case I'm missing something?

Reply:
I changed this to be crate visible only and behind the `cli` feature flag.

Reply:
Sorry to re-open this, but the command is now not included by default, but it's not hidden if building with the private feature flag `__graph_builder`.

Reply:
Sounds great. Thank you!